### ISSUE TITLE
Consolidate basic articles

### ISSUE THREAD

**Issue by krlmlr** [krlmlr/dm#192]
1. dm.Rmd
2. dm-class-and-basic-operations.Rmd
3. dm-setup.Rmd

- [ ] There's a lot of redundancy in 2 and 3, maybe merge into one text?
- [ ] 1 is the first text that potential users will see after the README if they are interested. It needs to highlight the most important topics and link to extended comprehensive articles. Currently it emphasizes some features that seem secondary.
- [ ] Review all other articles, ensure that they fit together and each concept is described only once in detail and linked from the other places where it's used
- [ ] Think about README, ensure that README is consistent with the rest.
- [ ] Do we need `index.md` (different README on GitHub and front page)?

**Comment by TSchiefer** [comment]
Also the README needs to be looked over once more (just realized that the description of the filtering is -- once more -- a bit off)
